src/test/java/com/verapdf/selenium/pages/BeforeTest.java

```diff
@@ -0,0 +1,29 @@
+package com.verapdf.selenium.pages;
+
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.concurrent.TimeUnit;
+
+public class BeforeTest {
+    public WebDriver driver = new ChromeDriver();
+
+    @BeforeMethod
+    public void beforeStart() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\User\\Documents\\verapdf-webapp-selenium-tests\\chromedriver.exe");
+        driver.get("https://verapdf.duallab.com/demo/new-job/files");
+        driver.manage().window().setSize(new Dimension(1044, 788));
+        driver.manage().timeouts().pageLoadTimeout(60 * 3, TimeUnit.SECONDS);
+        driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
+}
```

Comment:
Could you please update your idea settings to always add line endings to files:
![image](https://user-images.githubusercontent.com/5430498/82567421-04ff8900-9b86-11ea-8c2c-e8da2649e49e.png)

We should get rid of 
![image](https://user-images.githubusercontent.com/5430498/82567468-15afff00-9b86-11ea-9bb4-7c59bef5e60e.png)